plugin/restore/mysql/mysql.go

```diff
@@ -297,8 +405,8 @@ func getPITROldDatabaseName(database string, suffixTs int64) string {
 
 // SyncArchivedBinlogFiles syncs the binlog files between the instance and `binlogDir`,
 // but exclude latest binlog. We will download the latest binlog only when doing PITR.
-func (r *Restore) SyncArchivedBinlogFiles(ctx context.Context, instance *api.Instance, binlogDir string) error {
-	binlogFilesLocal, err := ioutil.ReadDir(binlogDir)
+func (r *Restore) SyncArchivedBinlogFiles(ctx context.Context) error {
+	binlogFilesLocal, err := ioutil.ReadDir(r.binlogDir)
 	if err != nil {
 		return err
 	}
```

Comment:
I may choose to add a `latest` flag to BinlogFile instead of creating 2 methods getBinlogFilesMetaOnServer and getLatestBinlogFileMeta

Reply:
It's simple and also avoids subtle timing issue that the latest binlog happens to get archived between calling 2 methods.

Reply:
I will handle it in another PR. It looks like it doesn't affect correctness.